apimanager/metrics/forms.py

```diff
@@ -218,22 +215,22 @@ class ConnectorMetricsForm(MetricsForm):
 class CustomSummaryForm(forms.Form):
     to_date = forms.DateField(
         label=_(TO_DATE),
-        widget=DatePickerInput(format='API_MANAGER_DATE_FORMAT'),
+        widget=DatePickerInput(format='%Y-%m-%d'),
```

Comment:
@Reena-cell Please use API_MANAGER_DATE_FORMAT; the issue here is that API_MANAGER_DATE_FORMAT is enclosed with single quote which represent as a string.